pkg/platform/kube/node.go

```diff
@@ -0,0 +1,35 @@
+package kube
+
+import (
+	"github.com/nuclio/nuclio/pkg/platform"
+
+	"k8s.io/api/core/v1"
+)
+
+type node struct {
```

Comment:
Why do we need a new type? Also - use 1.9 type aliases?

Reply:
`node` implements the `Node` interface, need a new type

Reply:
Also, currently we're 1.8